package.json

```diff
@@ -13,7 +13,7 @@
   },
   "scripts": {
     "test": "mocha",
-    "bundle": "browserify password.js showHide.js -o bundle.js"
+    "bundle": "browserify password.js showHide.js passwordGenerator.js -o bundle.js"
```

Comment:
No need to add it to the bundle. Since we only load it on the new html you're going to create

Reply:
I added it to the bundle to be able to check the code working. It wasn't possible without it.